### PR TITLE
misc bug fixes & small changes

### PR DESCRIPTION
**Pull request by Kyle-Kyle**

### CODE REVIEW

**Comment by Kyle-Kyle** [comment]
Do not merge yet. There is more to come.....

**Comment by Kyle-Kyle** [comment]
@zardus 
Hey Yan. I wonder what are these. And wtf is this line
`echo "$@" > /tmp/commands`
in https://github.com/angr/archr/blob/master/archr/arrows/shellphish_qemu/fire and https://github.com/angr/archr/blob/master/archr/arrows/emurrate/fire
And whether it is same to simply just remove them

**Comment by Kyle-Kyle** [comment]
And please me they are just for debugging and nothing actually uses them.

File: archr/targets/docker_target.py [hunk 1]
```diff
@@ -315,7 +314,6 @@ def check_in_docker() -> bool:
     with open("/proc/1/cgroup", "r") as f:
         return "docker" in f.read()
 
-
```
Comment (twizmwazin) [inline archr/targets/docker_target.py:318]: There should be two empty lines here

Comment (Kyle-Kyle) [reply archr/targets/docker_target.py:318]: OK

Comment (Kyle-Kyle) [reply archr/targets/docker_target.py:318]: wait I don't get it. Shouldn't the double empty lines be above `check_in_docker` not below?

Comment (twizmwazin) [reply archr/targets/docker_target.py:318]: There should be two above and two below. There should be two empty lines between each top-level statement.

Comment (Kyle-Kyle) [reply archr/targets/docker_target.py]: OK. changed

**Comment by Kyle-Kyle** [comment]
@twizmwazin Can you merge this?

**Pull request merged**